undef close macro from libepoll-shim at end of internal.h to avoid conflict with callbacks.close struct member

Index: glfw/internal.h
--- glfw/internal.h.orig
+++ glfw/internal.h
@@ -904,3 +904,7 @@ void _glfw_free_clipboard_data(_GLFWClipboardData *cd)
     } while (errno == EINTR)
 
 int _glfwApplyModifierRemap(int mods);
+// libepoll-shim defines close() as a macro; undef to avoid conflicts with struct members
+#ifdef close
+#  undef close
+#endif
